docs/bots-guide.md

```diff
@@ -109,12 +109,15 @@ You need:
     * Now, the bot can do its job on the streams you subscribed it to.
     * (In future versions of the API, this step may not be required).
 
-5. Run the bot.
+5. Run the bot. (There are two ways to run your bot.)
 
     * Run
       ```
       zulip-run-bot <bot-name> --config-file ~/.zuliprc
       ```
+      ```
+      zulip-run-bot <path/to/twibot.py> --config-file ~/.zuliprc
```

Comment:
Many people reading these docs won't know twibot, so it's best to use something more generic here, e.g. `path/to/bot.py`.

Reply:
oh no! that was an oversight. It's actually supposed to be just bot.py. Thanks for pointing it out.